client/base_client.go

```diff
@@ -145,7 +146,13 @@ func (c *baseClient) leaderLoop() {
 		case <-ctx.Done():
 			return
 		}
-
+		skipUpdateLeader := false
+		failpoint.Inject("skipUpdateLeader", func() {
+			skipUpdateLeader = true
```

Comment:
How about directly continue?

Reply:
we can't directly continue in this fuction.